lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -629,6 +629,7 @@
     import boto.ec2
     from boto.ec2.blockdevicemapping import BlockDeviceType, BlockDeviceMapping
     from boto.exception import EC2ResponseError
+    from boto.vpc import VPCConnection
```

Comment:
I believe this isn't needed - this was in there for a previous version of the fix IIRC.